src/style.css

```diff
@@ -44,19 +51,17 @@ body {
 }
 
 .is-active {
-  box-sizing: border-box;
-  color: #00acdc;
   margin-left: 0;
+  color: #00acdc;
 }
 
 .is-active::after {
   position: absolute;
-  top: 56px;
+  bottom: 0;
   left: 0;
-  height: 1px;
-  width: 34px;
+  height: 4px;
+  width: 37px;
```

Comment:
dont try to build it only for apple nav link, it should be reusable, use 100% width